src/canvas.common.ts

```diff
@@ -23,6 +23,8 @@ export const hardwareAcceleratedProperty = new Property<CanvasBase, boolean>({ n
 export const callDrawBeforeShapesProperty = new Property<CanvasBase, boolean>({ name: 'callDrawBeforeShapes', defaultValue: false, valueConverter: booleanConverter });
 export const densityProperty = new Property<CanvasBase, number>({ name: 'density', defaultValue:DEFAULT_SCALE, valueConverter: parseFloat });
 
+export * from './shapes';
+
```

Comment:
that i dont want :D
Can mess up tree shaking

Reply:
What alternative can we use to export shapes? Otherwise, they aren't available for Core applications.

Reply:
same thing. 
Your issue is the core xml not knowing them?
If so then i guess we need this. But kind of dont like it :D 
Almost sure it will break tree shaking. Let s try that and i ll try and see if it works with tree shaking

Reply:
> same thing.
> Your issue is the core xml not knowing them?

That's right.
I inspired this from @nativescript/core/index.ts exports. Let's hope it works. :D